source/content/guides/composer-convert.md

```diff
@@ -166,119 +157,130 @@ What makes your site code unique is your selection of contributed modules and th
+- The script does not resolve `composer.json` version problems.
 
-1. This script DOES NOT Resolve module stability.
+  - If there's a version conflict when you run `composer install`, you will need to resolve the conflicts yourself.
 
-   If you get the following error:
+- The script does not check for Drupal 9 compatibility.
+
+  - Use the [Upgrade Status](https://drupal.org/project/upgrade_status) module to check for Drupal 9 compatibility.
+
+- The script does not resolve module stability.
+
+  - If you get the following error, the module listed in the error (or its dependencies) does not meet compatibility requirements:
 
    ```none
    [InvalidArgumentException]
    Could not find a version of package drupal/MODULE_NAME matching your minimum-stability (stable). Require it with an explicit version constraint allowing its desired stability.
    ```
 
-   It means that one of the modules you are using (or its dependencies) are not stable. If there is not a stable version you can switch to, you may need to adjust the `minimum-stability` setting of `composer.json` to a more relaxed value, such as `beta`, `alpha`, or even `dev`. You can read more about `minimum-stability` [in the documentation for Composer](https://getcomposer.org/doc/04-schema.md#minimum-stability) itself.
+   If there is no stable version you can switch to, you may need to adjust the `minimum-stability` setting of `composer.json` to a more relaxed value, such as `beta`, `alpha`, or even `dev`. You can read more about `minimum-stability` in the [Composer documentation](https://getcomposer.org/doc/04-schema.md#minimum-stability).
 
-1. This script DOES NOT resolve patches.
+- The script does not resolve patches.
 
-   Many times a module will be "patched" or have a `.patch` file that fixes known issues before the fix is available in the downloaded version. This script does not attempt to resolve any patches.
+  Many times a module will be "patched" or have a `.patch` file that fixes known issues before the fix is available in the downloaded version. This script does not attempt to resolve any patches.
 
-To run the module migration script, cd to the root directory of your repository and run `bin/migrateModules.php`.
+If you still want to try it:
 
-You can see which modules were added by running `git diff composer.json`.
+1. Say the following out loud: "Pantheon is not responsible for what I am about to do."
 
-Then remove the `composer.lock` file (you can delete it. Composer will create a new one with the modules you just added) and run `composer install` and resolve any lingering version conflicts.
-</Alert>
+1. Create a directory called `bin` within your repository and copy the `migrateModules.php` file from [GitHub](https://github.com/pantheon-upstreams/drupal-project/pull/8/files#diff-c68470275758ca395b98d53ed258b63519435d492dd51531c4cf372814c6593e) into that directory.
```

Comment:
We should publish this script somewhere else. We could tell them to install via `composer require` if we published on Packagist.

I still think we should test and publish the docs without any scripts first, and then add these instructions as a follow-on improvement.